src/pages/Explorer/Category/Widget.svelte

```diff
@@ -0,0 +1,57 @@
+<script>
+  import Svg from 'webkit/ui/Svg/svelte'
+  import Category from './Category.svelte'
+  import ChartLayoutItem from '../Layouts/ChartLayoutItem.svelte'
+  import SocialTrend from '../Layouts/SocialTrend.svelte'
+  import WeeklyReport from '../Layouts/WeeklyReport.svelte'
+  import SheetsTemplate from '../Layouts/SheetsTemplate.svelte'
+
+  export let title = 'Recent Chart Layouts'
+  export let headerLeftIcon = 'info'
+  export let bgColor = 'var(--green-light-1)';
+  export let fill = 'var(--green)';
+  export let type
+  export let items = [
+    {
+      id: 0,
+      type: 'chart',
+      title: 'Protocol - Fundamental comparison (TVL, User, Develel)',
+      comments: 0,
+      votes: 0,
+      user: { username: 'test' },
+      assets: [
+        { slug: 'bitcoin' },
+        { slug: 'ethereum' },
+        { slug: 'bitcoin-cash' },
+        { slug: 'bitcoin' },
+      ],
+    },
+  ]
+</script>
+
+<Category {title} small {items}>
+  <div slot="icon" style="background-color: {bgColor}; fill: {fill}" class="$style.icon">
+    <Svg id={headerLeftIcon} w="16" />
+  </div>
```

Comment:
1. `$style` accessor is needed only for custom components;
2. Since all icons have this pattern `color[-light-1]`, style can be reduced to `style="fill:var(--{color}); background:var(--{color}-light-1)"`;
4. Just `icon` instead of `headerLeftIcon`;
5. Not all icons have `width=16`

Reply:
1. without `$style` it does not work (tested)